templates/secret-gitlab.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: gitlab-token
+  labels:
+    app: {{ .Values.name }}
+type: Opaque
+data:
+  token: {{ .Values.gitlab.token | b64enc | quote }}
```

Comment:
It might be good to make the secrets optional as they might not be available when users are doing a deployment but are already in the cluster.